### PR TITLE
Feat/#73 signup/login fix

### PR DESCRIPTION
**Pull request by cassjsdev**
**Pushed changes:**
- User should not be required to enter symbols at signup.
- When user signs up and is redirected to login, the username should be pre-filled.

### CODE REVIEW

**Comment by cassjsdev** [comment]
I edited the comment up top and removed WIP. I figured we can go ahead and merge this in time for the presentation. The only change is the user is not required to put in symbols at sign up. 

I will add the remaining (username prefill on login page after user signup) item to the kanban board as a future addition. Please review and approve. @MattiWheels @midCenturyDude7

File: app/templates/login.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <div class="form-group">
       <label for="username-input">Username</label>
       {% if username %}
+      <!-- {{ form.username(class="form-control bg-dark text-light border-0", type="text", id="username-input", placeholder="", required) }} -->
```
[the remark this answers is not in the document]
Comment (cassjsdev) [reply app/templates/login.html]: Note: Moving this feature (username prefill at login after user signup) to kanban board.

**Review by Xuehong-pdx — APPROVED** [review]
If I remember right, in an older version, after signup the user is redirected to the login page with the username pre-filled.  I checked the code and the logic is there, but it seems not being used.  I guess it is OK at this point.   In  user_svc.py line 23, the logical page to redirect to should be login.html, but somehow it is redirected to the dashboard.html.  It doesn't change the behavior since there is a safe guard in dashboard to redirect user to the login page if not logged in, but it should be changed back to redirect to the login page, more logical that way.  Other than that, the app seems be working fine. Good job.

**Comment by cassjsdev** [comment]
> If I remember right, in an older version, after signup the user is redirected to the login page with the username pre-filled. I checked the code and the logic is there, but it seems not being used. I guess it is OK at this point. In user_svc.py line 23, the logical page to redirect to should be login.html, but somehow it is redirected to the dashboard.html. It doesn't change the behavior since there is a safe guard in dashboard to redirect user to the login page if not logged in, but it should be changed back to redirect to the login page, more logical that way. Other than that, the app seems be working fine. Good job.

Did you do a pull from 'feat/#73-signup-login-fix'? Try deletng DB and clear cache completely, or delete and reclone project folder. I just did an entirely new clone of the project folder and deleted my old one from my local machine to confirm. Seems to be acting okay from my end. It's redirecting me to login after entering sign up details.

**Comment by Xuehong-pdx** [comment]
it does redirect to login because dashboard does that if user is not logged
in.  but the current code added an extra step by directing user to
dashboard instead of login on line 23.  want me to push the code up?

**Review by MattiWheels — APPROVED** [review]
Simple but effective! This hasn't broken anything on my end (that I am aware of through manual testing).

File: app/services/user_svc.py [hunk 2]
```diff
@@ -19,8 +19,8 @@ def signup():
             db.session.add(user)
             db.session.commit()
             flash(f'Welcome {user.username}! Please login.', 'notify')
-            # lform.username.data = user.username
-            return redirect(url_for('main_bp.dashboard'))
+            lform.username.data = user.username
```
Comment (cassjsdev) [inline app/services/user_svc.py:22]: I completely forgot to ask about this awhile back. I remember now this line of code was commented out for some reason. Good job @Xuehong-pdx !

**Pull request merged**